### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_AI4W: The “Get the latest release each day.” and “Read more in the blog...” texts are provided with unnecessary heading elements.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2795]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated : MAS 2.4.6

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate through the "Download Insiders" screen.
5.	Navigate to “Get the latest release each day.” and “Read more in the blog...” texts.
6.	Observe if these texts are provided with heading or not.

**Actual:**
The “Get the latest release each day.” and “Read more in the blog...” texts are provided with heading elements of level 3 which is incorrect as these texts are defining any specific section on the screen.

**Expected:**
when screen reader users navigate to the “Get the latest release each day.” and “Read more in the blog...” texts, they should not be announced as heading elements.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If some text on the screen are provided with heading elements unnecessarily then the screen reader users might not understand the screen content structure.

**MAS Reference:**
MAS 2.4.6 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}) 

**Attachment for Reference:**
![Code visualstudio com_DownloadInsiders_K4W Unnecessary heading elements](https://user-images.githubusercontent.com/52036772/60008717-2d3ee900-9692-11e9-8f7a-23fb6169f56f.png)

**Comment by octref** [comment]
> which is incorrect as these texts are defining any specific section on the screen

If you meant "aren't defining any specific section on the screen", I don't think it violates the MAS guideline, which states:

> When headings and labels are used, they must describe the topic or purpose of the element or content that they pertain to.

“Get the latest release each day.” describes the purpose of this page.
“Read more in the blog...” describe the purpose of the "blog" link.
